### ISSUE TITLE
Regression: first item in ordered plugin dialog is empty

### ISSUE THREAD

**Issue by matterhorn103** [OpenChemistry/avogadrolibs#1632]
**Avogadro version: (please complete the following information from the About box):**
 - Avogadrolibs: 1.99.0
 - Qt: 5.15.2

**Desktop version: (please complete the following information):**
 - OS: Linux AppImage, Windows, macOS
 - Version: various
 - Compiler: for self-compiled build, clang

**Describe the bug**
The bug that I described and fixed [here](https://github.com/OpenChemistry/avogadrolibs/pull/1523) seems to be back.

Weirdly, it doesn't occur on my personal build of Avogadro from the latest codebase. However, it does in the 1.99 AppImage, I get the impression it's also [happening on Windows](https://github.com/matterhorn103/avo_xtb/issues/5), and @JGrantHill also reports it happening on macOS (ARM).

**To Reproduce**
1. Install the avo_xtb plugin
2. Open the "Configure..." menu or anything else that has a dialog e.g. Run or Conformers
3. Observe that first field is always empty, even after entering a path and saving the config, or after adding the path manually to `Avogadro/commands/avo_xtb/plugin.json`

**Comment by matterhorn103** [comment]
Still bizarrely remains the case that the version of Avogadro that I have compiled myself doesn't have this issue, but all other versions apparently do.

Could this be a bug that was fixed in very recent Qt5 versions? I compiled against 5.15.15, whereas the nightly AppImage is compiled against 5.15.2.

**Comment by ghutchis** [comment]
That's certainly possible. Is the qt6 version able to run the plugins? If so, does that fix the bug?

My feeling is that it's probably better to push ahead with migrating to qt6 sooner.

**Comment by matterhorn103** [comment]
> Is the qt6 version able to run the plugins?

No, sadly. That's actually the only thing I've been able to spot that doesn't work in my Qt6 build.

**Comment by matterhorn103** [comment]
`QRegularExpression` has 38 instances in `avogadrolibs`, many of them in files related to the plugins, so I'll prioritize porting those over the next week or so.

**Comment by ghutchis** [comment]
@matterhorn103 - why don't you post something on the forum. Maybe a few others would be willing to help with porting.

IIRC some of that has to do with context highlighting in the input generators, which isn't so crucial. That could be `#ifdef` for qt5-only for now.

**Comment by ghutchis** [comment]
Just to check in - does this still happen with the current HEAD?

**Comment by matterhorn103** [comment]
Yep, this is a bug in Qt 5.15.2 that was fixed in some later release.

If I build from source against Qt 5.15.15 or Qt 6 it's fine, but the AppImage and nightly binaries for Windows/Mac still have the issue because 5.15.2 is the only Qt5 binary available.

**Comment by matterhorn103** [comment]
With #1908 now merged, all the distributed Linux builds of Avogadro are built against Qt6, so this issue will be fixed on Linux in the upcoming 1.100 release. Migration to Qt6 on Windows and Mac has been postponed until after 1.100, in time for the subsequent (likely summer) release, so the bug will unfortunately still raise its head on those platforms for now. Leaving this open until then.

**Comment by ghutchis** [comment]
Is this still a bug now that everything is migrated to Qt6? Can we close it?

**Comment by ghutchis** [comment]
I'll repeat - is this still a bug @matterhorn103

**Comment by matterhorn103** [comment]
Sorry! Yes, it seems to be fixed in all the Linux versions. @brockdyer03, would you mind checking on Windows?

**Comment by brockdyer03** [comment]
> Sorry! Yes, it seems to be fixed in all the Linux versions. [@brockdyer03](https://github.com/brockdyer03), would you mind checking on Windows?

I'm installing the latest kernel for Fedora 43 right now. Once that is installed I'll boot into Windows and give it a test!

**Comment by brockdyer03** [comment]
Good to go on Windows, feel free to close this.
